packages/ui-components/src/Input/TextTypeInput.tsx

```diff
@@ -24,26 +25,11 @@ export interface MultilineProps
   suffix?: JSX.Element;
 }
 
-// the following props are used to implement {Email, Password, Number}Input
-// it is recommended to use those components instead of passing an "email" type to TextInput
-interface CustomProps extends React.InputHTMLAttributes<HTMLInputElement> {
-  ariaLabel?: string;
-  ariaLabelledBy?: string;
-  forgotPasswordLinkElement?: JSX.Element;
-  // this prop is only used for internal implementation
-  // whether the password exists or not, should be indicated by New/Existing PasswordInput
-  existingPassword?: boolean;
-}
```

Comment:
Moved to the `./EmailPassword` file as components have been refactored such that these props are only necessary for `PasswordInput`.